src/homework6/dataType/ListOfMethods.java

```diff
@@ -0,0 +1,77 @@
+package homework6.dataType;
+
+import java.util.Random;
+
+public class ListOfMethods {
+    void printThreeWords() {
+        System.out.println("Orange" + "\n" + "Banana" + "\n" + "Apple");
+    }
+
+    void checkSumSign(int a, int b) {
+        if ((a + b) >= 0) {
+            System.out.println("Sum is positive");
+        } else {
+            System.out.println("Sum is negative");
+        }
+    }
+
+    void printColor(int value) {
+        if (value <= 0) {
+            System.out.println("red");
+        } else if (value >= 100) {
+            System.out.println("yellow");
+        } else {
+            System.out.println("green");
+        }
+    }
+
+    void compareNumbers(int a, int b) {
+        if (a >= b) {
+            System.out.println("a >= b");
+        } else {
+            System.out.println("a < b");
+        }
+    }
+
+    public boolean sumTenFromTwenty(int a, int b) {
+        if (((a + b) > 10) && ((a + b) <= 20)) {
```

Comment:
better to add separate temp var for sum, because it calculating sum twice. 

better dont use if in case you only true/false 
I mean it can be only 
`return ((a + b) > 10) && ((a + b) <= 20);`